src/evoKnowledgeShare/evoKnowledgeShare.Backend/Services/NoteService.cs

```diff
@@ -127,7 +151,48 @@ public async Task<IEnumerable<Note>> AddRangeAsync(IEnumerable<Note> notes)
         /// <param name="note"></param>
         /// <returns>A <see cref="Note"/> if modified</returns>
         /// <exception cref="KeyNotFoundException"></exception>
-        public Note Update(Note note) => myRepository.Update(note);
+        public Note Update(Note note, string mdRaw, int incrementSize) {
```

Comment:
van beépített `System.Version`, használjuk azt
`incrementSize` helyett ezt engedjük megmondani a kliensnek, hogy mi legyen az új
legyen ellenőrzés arra, hogy ne hozzunk létre olyat ami már van, ne hagyjunk ki verziót pl:
- 1.0.0 -> 1.0.1 ok
- 1.0.0 -> 1.0.2 nem ok

Reply:
Adom.